spaceship.zsh

```diff
@@ -405,7 +405,9 @@ spaceship_dir() {
 }
 
 # GIT BRANCH
-# Show current git brunch using git_current_status from Oh-My-Zsh
+# Show current git brunch using git_current_branch from Oh-My-Zsh
+# Reference:
+#   https://github.com/robbyrussell/oh-my-zsh/blob/master/lib/git.zsh
```

Comment:
Duplicate of [L425](https://github.com/denysdovhan/spaceship-zsh-theme/pull/145/files#diff-576119bcd2b03841164132b326cfca94R425).